tests/e2e/support/api/token/utils.ts

```diff
@@ -0,0 +1,88 @@
+import { TokenEnvironmentFactory } from '../../environment'
+import { config } from '../../../config'
+import fetch from 'node-fetch'
+import { User } from '../../types'
+const logonUrl = config.backendUrl + '/signin/v1/identifier/_/logon'
+const redirectUrl = config.backendUrl + '/oidc-callback.html'
+const tokenUrl = config.backendUrl + '/konnect/v1/token'
+const getAuthorizedEndPoint = async (user) => {
+  const logonResponse = await fetch(logonUrl, {
+    method: 'POST',
+    headers: {
+      'Kopano-Konnect-XSRF': '1',
+      Referer: config.backendUrl,
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      params: [user.id, user.password, '1'],
+      hello: {
+        scope: 'openid profile email',
+        client_id: 'web',
+        redirect_uri: redirectUrl,
+        flow: 'oidc'
+      }
+    })
+  })
+  if (logonResponse.status !== 200) {
+    throw new Error(`logonResponse.status is ${logonResponse.status}, expected 200`)
+  }
+  const cookies = logonResponse.headers.raw()['set-cookie']
+  const data = await logonResponse.json()
+  const authorizedUrl = data.hello.continue_uri
+  return [authorizedUrl, cookies]
+}
+
+const getAuthorizedToken = async (user, continueUrl, cookies) => {
+  const params = new URLSearchParams({
+    client_id: 'web',
+    prompt: 'none',
+    redirect_uri: redirectUrl,
+    response_mode: 'query',
+    response_type: 'code',
+    scope: 'openid profile offline_access email'
+  })
+  const authorizeResponse = await fetch(`${continueUrl}?${params.toString()}`, {
+    method: 'GET',
+    redirect: 'manual',
+    headers: {
+      Cookie: cookies
+    }
+  })
+  if (authorizeResponse.status !== 302) {
+    throw new Error(`logonResponse.status is ${authorizeResponse.status}, expected 302`)
+  }
+  const locationHeader = authorizeResponse.headers.get('location')
+  const urlParams = new URLSearchParams(new URL(locationHeader).search)
+  return urlParams.get('code')
+}
+
+const getToken = async (user, code, cookies) => {
+  return await fetch(tokenUrl, {
+    method: 'POST',
+    headers: {
+      Cookie: cookies
+    },
+    body: new URLSearchParams({
+      client_id: 'web',
+      code: code,
+      redirect_uri: redirectUrl,
+      grant_type: 'authorization_code'
+    })
+  })
+}
+
+export const getAccessAndRefreshTokenViaApi = async ({ user }: { user: User }) => {
```

Comment:
```suggestion
export const getAccessToken = async ({ user }: { user: User }) => {
```

Reply:
or because the function doesn't return
```suggestion
export const setAccessToken = async ({ user }: { user: User }) => {
```

Reply:
But this stores both Access and Refresh token.